contrib/verify-commits/verify-commits.py

```diff
@@ -83,17 +83,17 @@ def main():
     dirname = os.path.dirname(os.path.abspath(__file__))
     print("Using verify-commits data from " + dirname)
     with open(dirname + "/trusted-git-root", "r", encoding="utf8") as f:
-        verified_root = f.read().splitlines()[0]
+        verified_root = set(f.read().splitlines())[0]
```

Comment:
Have you even tested this? You can't subscript index into a set, and it makes no sense.

Reply:
This is not even an equivalent expression. When you convert a list to a set, the order of the lines is lost. Therefore, accessing the first element with [0] could give you any of the lines randomly, not necessarily the first line from the original file. In this case you needed index 0 ( root ).

Reply:
> This is not even an equivalent expression. When you convert a list to a set, the order of the lines is lost. Therefore, accessing the first element with [0] could give you any of the lines randomly, not necessarily the first line from the original file. In this case you needed index 0 ( root ).

You're right. It makes no sense to use a set in this context. Thanks for pointing this out